salt/modules/rpm.py

```diff
@@ -168,14 +168,34 @@ def verify(*packages, **kwargs):
               'g': 'ghost',
               'l': 'license',
               'r': 'readme'}
+    options_map = {
+        'nocaps': '--nocaps',
+        'nodeps': '--nodeps',
+        'nodigest': '--nodigest',
+        'nofiledigest': '--nofiledigest',
+        'nofiles': '--nofiles',
+        'nogroup': '--nogroup',
+        'nolinkto': '--nolinkto',
+        'nomode': '--nomode',
+        'nomtime': '--nomtime',
+        'nordev': '--nordev',
+        'noscripts': '--noscripts',
+        'nosignature': '--nosignature',
+        'nosize': '--nosize',
+        'nouser': '--nouser'
+    }
     ret = {}
     ignore_types = kwargs.get('ignore_types', [])
+    verify_options = kwargs.get('verify_options', [])
+    cmd = ['rpm']
+    for option in verify_options:
+        cmd.append(options_map.get(option, ''))
```

Comment:
There's certainly more than one way to do it. I like having the map because it helps scope and document what options this would work with.

I'm pretty sure I tested passing a bad option, and it didn't blow anything up. I'm happy to test it again if you like.

String vs list, I was mostly just copying the implementation of the `ignore_types` option. So perhaps that, too, should split if it is a string?